docker/management-ui/Dockerfile

```diff
@@ -14,14 +14,11 @@
 # limitations under the License.
 #
 
-FROM nginx:1.23-alpine
+FROM graviteeio/nginx:1.22
```

Comment:
gio tag not consistent with the nginx tag used to build this [base image](https://github.com/gravitee-io/gravitee-docker/blob/master/images/nginx/Dockerfile#L13) 

Should be graviteeio/nginx:1.23

Reply:
upgraded